universal-login-relayer/test/integration/ethereum/ContractService.test.ts

```diff
@@ -0,0 +1,66 @@
+import {expect} from 'chai';
+import {TEST_ACCOUNT_ADDRESS, TEST_GAS_PRICE, ETHER_NATIVE_TOKEN, DEFAULT_GAS_LIMIT, OperationType} from '@universal-login/commons';
+import {WalletContractService} from '../../../src/integration/ethereum/WalletContractService';
+import {BlockchainService, messageToSignedMessage} from '@universal-login/contracts';
+import {createMockProvider, getWallets} from 'ethereum-waffle';
+import {Beta2Service} from '../../../src/integration/ethereum/Beta2Service';
+import {Contract, Wallet, utils} from 'ethers';
+import createWalletContract from '../../testhelpers/createWalletContract';
+import {getTestSignedMessage} from '../../testconfig/message';
+
+describe('WalletContractService', () => {
```

Comment:
int or unit?

Reply:
I would consider writing only UNIT tests for this class, integration will be covered anyway in higher lever services.